pkg/apiserver/apiserver_test.go

```diff
@@ -931,7 +904,7 @@ func TestList(t *testing.T) {
 			legacy:    true,
 		},
 		{
-			url:       "/" + prefix + "/" + testGroupVersion.Group + "/" + testGroupVersion.Version + "/simple?namespace=other&labels=a%3Db&fields=c%3Dd",
+			url:       "/" + prefix + "/" + testGroupVersion.Group + "/" + testGroupVersion.Version + "/simple?namespace=other&labelSelector=a%3Db&fieldSelector=c%3Dd",
```

Comment:
Sorry I'm not familiar with the background of this PR. Is `labels=a%3Db&fields=c%3Dd` a query sent by old clients? If so, we probably should keep the original line to test the backward compatibility.